### PR TITLE
[WIP] Remove bootstrap

### PR DESCRIPTION
**Pull request by akgupta89**
Completely removes Bootstrap. This kills certain styling that was associated with Bootstrap Cards and Forms, that wasn't carried over, so we need to either modify this PR to add the styling back or create a follow up PR to this.

Screen Shot 2019-07-31 at 9 00 25 PM

e.g. this looks terrible and needs to be restyled with material-ui

### CODE REVIEW

**Comment by akgupta89** [comment]
We can wait to merge this when we have implemented material-ui styling or merge it in and tackle that later, up to you guys

**Comment by hathix** [comment]
i would rather not merge this until the stylistic losses are addressed. That is, this PR should include styling fixes to make sure cards still look good.

**Comment by akgupta89** [comment]
Screen Shot 2019-08-07 at 8 07 32 PM
Screen Shot 2019-08-07 at 8 07 49 PM
Screen Shot 2019-08-07 at 8 08 01 PM

Carded and padded everything

**Comment by exxonvaldez** [comment]
Looks good.  Probably better not to commit package-lock.json -- from what I've seen, this just creates conflicts for everybody else.

File: frontend/src/components/InfoIntervalsOfDay.jsx [hunk 1]
```diff
@@ -102,68 +101,70 @@ class InfoIntervalsOfDay extends Component {
...
-                  label="Average"
-                  value={InfoIntervalsOfDay.AVERAGE_TIME}
-                  checked={this.state.selectedOption === InfoIntervalsOfDay.AVERAGE_TIME}
-                  onChange={this.handleOptionChange} 
+              <CardContent>
+                <CardHeader>Performance by Time of Day</CardHeader>
+                <FormControl>
+                <div className="controls">
+                  <FormControlLabel
+                    control={<Checkbox
```
Comment (exxonvaldez) [inline frontend/src/components/InfoIntervalsOfDay.jsx]: Just noticed these are checkboxes, can you switch them to Radio? https://material-ui.com/components/radio-buttons/

**Pull request merged**